api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -1,95 +1,127 @@
 # -*- coding: utf-8 -*-
-import random
-
-import mock
-
 from datetime import datetime
-
-from nose.tools import *  # flake8: noqa
-
-from rest_framework import exceptions
+import mock
+import pytest
+import random
 
 from api.base.exceptions import Conflict
 from api.base.settings.defaults import API_BASE
 from api.nodes.serializers import NodeContributorsCreateSerializer
-
 from framework.auth.core import Auth
-
-from tests.base import ApiTestCase, capture_signals, fake
+from osf.models import NodeLog
 from osf_tests.factories import (
     ProjectFactory,
     AuthUserFactory,
     UserFactory,
-    UnconfirmedUserFactory
+    UnconfirmedUserFactory,
 )
-from tests.utils import assert_logs
-
-from osf.models import NodeLog
+from rest_framework import exceptions
+from tests.base import capture_signals, fake
+from tests.utils import assert_latest_log, assert_items_equal
 from website.project.signals import contributor_added, unreg_contributor_added, contributor_removed
 from website.util import permissions, disconnected_from_listeners
 
 
-class NodeCRUDTestCase(ApiTestCase):
+@pytest.fixture()
+def user():
+    return AuthUserFactory()
 
-    def setUp(self):
-        super(NodeCRUDTestCase, self).setUp()
-        self.user = AuthUserFactory()
-        self.user_two = AuthUserFactory()
+@pytest.mark.django_db
+class NodeCRUDTestCase:
 
-        self.title = 'Cool Project'
-        self.new_title = 'Super Cool Project'
-        self.description = 'A Properly Cool Project'
-        self.new_description = 'An even cooler project'
-        self.category = 'data'
-        self.new_category = 'project'
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
 
-        self.public_project = ProjectFactory(title=self.title,
-                                             description=self.description,
-                                             category=self.category,
-                                             is_public=True,
-                                             creator=self.user)
+    @pytest.fixture()
+    def user_two(self):
+        return AuthUserFactory()
 
-        self.public_url = '/{}nodes/{}/'.format(API_BASE, self.public_project._id)
+    @pytest.fixture()
+    def title(self):
+        return 'Cool Project'
 
-        self.private_project = ProjectFactory(title=self.title,
-                                              description=self.description,
-                                              category=self.category,
-                                              is_public=False,
-                                              creator=self.user)
-        self.private_url = '/{}nodes/{}/'.format(API_BASE, self.private_project._id)
+    @pytest.fixture()
```

Comment:
Are these title and description fixtures used in more than one place? Might be overuse of fixtures. Not a blocker, though.